arangod/Cluster/ClusterInfo.cpp

```diff
@@ -2445,28 +2509,32 @@ int ClusterInfo::ensureIndexCoordinatorInner(std::string const& databaseName,
   VPackSlice indexes = collection.get("indexes");
   if (indexes.isArray()) {
     auto type = slice.get(arangodb::StaticStrings::IndexType);
+
     if (!type.isString()) {
-      return setErrormsg(TRI_ERROR_INTERNAL, errorMsg);
+      return Result(TRI_ERROR_INTERNAL);
```

Comment:
if possible, we should be a bit more specific here and add a descriptive error message, e.g. `expecting string for index "type" attribute`. Otherwise it is a nightmare finding out what triggered the internal error.
```suggestion
      return Result(TRI_ERROR_INTERNAL, "expecting string value for \"type\" attribute");
```

Reply:
ok, I'll add that